src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets

```diff
@@ -122,6 +134,16 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!--
+    ============================================================
+                                        AddTransitiveFrameworkReferences
+
+    Adds FrameworkReference items for shared frameworks which weren't directly referenced,
+    but were referenced transitively via a project or package reference.  NuGet writes these
+    to the assets file, and the ResolvePackageAssets target adds them to the TransitiveFrameworkReference
+    item.  Here, we add them to FrameworkReference if they aren't already referenced.
+    ============================================================
+    -->
   <Target Name="AddTransitiveFrameworkReferences" AfterTargets="ResolvePackageAssets"
```

Comment:
Will these appear as "ResolvedFrameworkReference"?

Reply:
Would it be a problem if you end up with ResolvedFrameworkReference items that didn't have a corresponding FrameworkReference when you evaluated?  Would you just ignore them or would it be a warning or error?

Reply:
We will ignore them.